sdk/ai/ai-inference-rest/test/public/utils/recordedClient.ts

```diff
@@ -41,3 +43,13 @@ export async function createModelClient(
   const credential = new AzureKeyCredential(apikey);
   return createClient(endpoint, credential, recorder?.configureClientOptions(options ?? {}));
 }
+
+export async function createEmbeddingsClient(
```

Comment:
Do we need this function? seems like `createModelClient` does the same job, no?

Reply:
We add a param telling whether to use an embedding endpoint or chatCompletions endpoint, is that what you're suggesting?

Reply:
Oh is the endpoint locked to the feature itself? Is it because of deployment ID being part of the path? You may consider having something like this so you don't end up with duplicate code: https://github.com/Azure/azure-sdk-for-js/blob/d367b3b142389ce21d7abbe53b2d443698cc2070/sdk/openai/openai/test/public/utils/recordedClient.ts#L59-L93

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-js/pull/30379/commits/e0ddee4ab5831f4bdaf4b786154c9d9099c0424f)